draft-ietf-dnsop-generalized-notify.md

```diff
@@ -251,10 +171,10 @@ the generalized notifications are a new proposal with no prior
 deployments, there is an opportunity to avoid repeating mistakes.
 
 The DSYNC record type also provides a cleaner solution for bundling all
-the new types of notification signaling in an RRset, like:
+the new types of notification signaling in one RRset, like:
 
-    parent.         IN DSYNC  CDS     1  59   scanner.parent.
-    parent.         IN DSYNC  CSYNC   1  59   scanner.parent.
+    IN DSYNC  CDS     1  59   scanner.example.net.
```

Comment:
I agree with the reviewer that it would be better to indicate the scheme via a mnemonic rather than a number. I've changed my code to parse/present schemes a la 1=NOTIFY and 2=UPDATE. I.e. for the example below:

child._dsync.parent.    IN DSYNC  CSYNC  NOTIFY  59 scanner.example.net.

Reply:
While I'm OK with that, I don't know why you "agree with the reviewer"; I can't find this suggestion in the [review](https://mailarchive.ietf.org/arch/msg/dnsop/PxPDGSbOIPpX320CwTLdEZHlvho/).

I'm wondering if it would be better to keep changes minimal this time, and introduce this in the next step. What do you think?